CorsixTH/Src/th_lua.h

```diff
@@ -277,8 +277,8 @@ template <> struct luaT_classinfo<string_proxy> {
     static inline const char* name() {return "StringProxy";}
 };
 
-struct THLfsExt;
-template <> struct luaT_classinfo <THLfsExt> {
+class lfs_ext;
```

Comment:
looks very short compared to the other names

Reply:
Extensions to luafilesystem. Ok I will change this to be clearer

Reply:
There doesn't seem to be anything "Lua specific" in lfs (eg special handling of lua files or so), just regular fs access for lua. Drop the "lua" part?

Reply:
luafilesystem is the name of a 3rd party module. This is an extension to make it play nicer with Windows.

Reply:
Keeping lfs, that's luafilesystem's internal name. e.g. `lfs.dir(...)`, `lfs.attributes(...)`